test/ttmlir/Conversion/StableHLOToTTIR/stablehlo_to_ttir_TransposeOp.mlir

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+#any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
+module @jit_transpose attributes {} {
+  func.func public @test_transpose(%arg0: tensor<64x128xf32>) -> tensor<128x64xf32> {
+    %0 = stablehlo.transpose %arg0, dims = [1,0] : (tensor<64x128xf32>) -> tensor<128x64xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
+    // CHECK: %[[C:.*]] = "ttir.transpose"[[C:.*]]
+    return %0 : tensor<128x64xf32>
+  }
+}
```

Comment:
Please rename this file to `transpose_op.mlir` to be consistent with other test files (I've renamed remaining files in GH-527)

Reply:
Sorry for being picky, but you've left the `stablehlo_to_ttir_` prefix in the name :) No need for that now since we've moved those files into separate folder `StableHLOToTTIR`.